src/components/menu/menu.ts

```diff
@@ -205,6 +208,8 @@ export class Menu implements RootNode {
   private _events: UIEventManager;
   private _gestureBlocker: BlockerDelegate;
   private _isPane: boolean = false;
+  private _side: side = 'start';
```

Comment:
@manucorporat I can only guess that this PR derailed a bit, and went through many changes that some stuff got left behind..
I thing I will fork this, and create a new cleaner PR.

---

Can you please comment on the tests? How can you access `Platform` / how can you run specific tests on different directions?